tests/scripts/thread-cert/thread_cert.py

```diff
@@ -483,14 +483,19 @@ def _parse_params(self, params: Optional[dict]) -> dict:
 
         if params.get('is_bbr') or params.get('is_otbr'):
             # BBRs must use thread version 1.2
-            assert params.get('version', '1.2') == '1.2', params
-            params['version'] = '1.2'
+            version = params.get('version', '1.2')
+            assert version != '1.1', params
+            params['version'] = version
             params.setdefault('bbr_registration_jitter', config.DEFAULT_BBR_REGISTRATION_JITTER)
         elif params.get('is_host'):
             # Hosts must not specify thread version
             assert params.get('version', '') == '', params
             params['version'] = ''
 
+        # use 1.3 node for 1.2 tests
+        if 'version' in params and params['version'] == '1.2':
+            params['version'] = '1.3'
+
```

Comment:
```suggestion
```

the code looks strange.

Reply:
This is to handle cases where `version` was set to `1.2` or the `version` key does not exist at all. Do you have a suggestion to improve this?